rel/i18n/emqx_slow_subs_schema.hocon

```diff
@@ -1,18 +1,23 @@
 emqx_slow_subs_schema {
 
 enable.desc:
-"""Enable this feature"""
+"""Enable Slow Subscriptions"""
 
 expire_interval.desc:
-"""The eviction time of the record, which in the statistics record table"""
+"""The expiration time of the slow subscription record, if the record is not updated within the expiration time, then the record will be deleted."""
 
 stats_type.desc:
-"""The method to calculate the latency"""
+"""Message latency calculation method: <br/>
+- whole: The time from when the message arrives at the EMQX (the EMQX gets the message from the receive buffer) until the message completes delivery. <br/>
+- internal: The time from when the message arrives at the EMQX (the EMQX gets the message from the receive buffer) to when the message begins to be delivered (the EMQX attempts to write the message to the send buffer). <br/>
+- response: The time from the start of message delivery to the completion.<br/>
+Note: The completion delivery time refers to the time when QoS 1 and 2 messages complete the MQTT message response process, i.e., the time when QoS 1 message receives the PUBACK packet and QoS 2 message receives the PUBCOMP packet. <br/>
+Since there is no response packet for QoS 0 message, the completion delivery time of the QoS 0 message will be replaced by the time when the message starts to be delivered. Therefore, when using the `response` method to calculate the latency, the latency of a QoS 0 message will always be equal to 0."""
```

Comment:
If i remember correctly, it should be when the message is put to the socket send buffer. So it might not always be 0.
please check the code to confirm.

Reply:
I checked the code, EMQX doesn't set `deliver_begin_at` header for qos 0 messages, so `Now - Now` always be 0.

```
calc_timespan(response, Msg, Now) ->
    Begin = emqx_message:get_header(deliver_begin_at, Msg, Now),
    Now - Begin.
```